bicep/deploy.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+
+# === Configuration ===
+AKS_RG_NAME="POCIMPINFRG1401"
+LOCATION="uksouth"
+#DNS_PREFIX="ipaffsaks"
+#ADMIN_USERNAME="ipaffsadmin"
+#SSH_KEY_PATH="$HOME/defra/ipaffs-infra/bicep/aks_id_rsa.pub"
+
+# === Resource Group for actual AKS and network resources ===
+echo "🔧 Ensuring AKS resource group exists: $AKS_RG_NAME"
+az group create --name $AKS_RG_NAME --location $LOCATION >/dev/null
+echo "✅ Resource group '$AKS_RG_NAME' is ready."
+
+# === Deploy main.bicep which triggers network + AKS module ===
+echo "🚀 Deploying Bicep templates to $AKS_RG_NAME..."
+<<'END'
+az deployment group create \
+  --resource-group $AKS_RG_NAME \
+  --template-file main.bicep \
+  --parameters dnsPrefix=$DNS_PREFIX \
+               linuxAdminUsername=$ADMIN_USERNAME \
+               sshRSAPublicKey="$(cat $SSH_KEY_PATH)" \
+               location=$LOCATION \
+               aksResourceGroupName=$AKS_RG_NAME \
+  --output json | tee deployment-output.json
+END
+
+az deployment group create \
+  --resource-group $AKS_RG_NAME \
+  --template-file main.bicep \
+  --parameters location=$LOCATION \
+  --output json | tee deployment-output.json
+
+echo "✅ Bicep deployment initiated."
+
+# === Optional: Wait for AKS readiness ===
+AKS_NAME="pocimpinfaks001"
```

Comment:
As above, let's uppercase names for consistency

```suggestion
AKS_NAME="POCIMPINFAK1401"
```

Reply:
Agree. I will probably kill all the .sh files periodically as going to do them on pipeline yaml and via from main.bicep modules. Those were mainly temp wrapper scripts I created for me to deploy without pipeline.